packages/node_modules/@webex/plugin-meetings/src/constants.js

```diff
@@ -8,6 +8,7 @@
 export const AUDIO = 'audio';
 export const AUDIO_BANDWIDTH_LIMIT = '64000';
 export const AUDIOINPUT = 'audioinput';
```

Comment:
`AUDIO_INPUT`

Reply:
i was copying some other one that didn't space, it, sure